package.json

```diff
@@ -21,6 +21,7 @@
     },
     "homepage": "https://github.com/Power-Components/livewire-powergrid#readme",
     "dependencies": {
+        "flatpickr": "^4.6.13",
```

Comment:
flatpickr should not be loaded inside the powergrid. It has to be loaded from the outside

Reply:
Hello,
when i ran the "npm run build" command for the first time, i got the error visible in the screenshot

![image](https://user-images.githubusercontent.com/4957015/168458386-3bcbbf2a-4b26-4f7e-9ec8-01c3fd98e5e8.png)

I tried to imagine that some dependency was missing. In fact, the build does not happen without adding the flatpickr library. 

Do you think it is appropriate to leave flatpickr but put it in the devDependencies or remove it?

Reply:
Put it as devDependencies, no problem

Reply:
Done! :sweat_smile:   
Let me know if now is ok